app/views/admin/badge_achievements/award.html.erb

```diff
@@ -1,21 +1,21 @@
-<main>
-  <h2>Award Badges</h2>
-  <div class="alert alert-warning"><strong>Warning: Usernames are case-sensitive and must be all lowercase! This task will fail silently when supplied with incorrect
-    usernames.<strong></div>
-<%= form_with(url: admin_badge_achievements_award_badges_path, local: true) do |f| %>
-<div class="form-group">
-  <%= f.label :badge, "Badge:" %>
-  <%= f.select("badge", @all_badges.map { |badge| [badge.title, badge.slug] }, include_blank: true, class: "form-control") %>
-</div>
-<div class="form-group">
-  <%= f.label :usernames, "Usernames (Comma Delimited, All Lowercase, No @ Symbol Required In Front Of Usernames)*" %>
-  <%= f.text_area :usernames, placeholder: "username1, username2, username3", required: true, size: "40x10", class: "form-control" %>
-</div>
-<div class="form-group">
-  <%= f.label :message_markdown, "Override Default Message (Supports Markdown)" %>
-  <%= f.text_area :message_markdown, size: "40x10", class: "form-control" %>
-</div>
-<%= f.submit "Award Badges", class: "btn btn-primary mb-4" %>
-<%= link_to "Back to Badge Achievements", admin_badge_achievements_path, class: "btn btn-primary float-right" %>
-<% end %>
+<h2 class="fs-2xl s:fs-3xl mb-6">Award Badges</h2>
```

Comment:
IIRC `<main>` was added by @jacobherrington to fix some a11y issues, is it okay if it disappears? Same for all the other pages

Reply:
@rhymes, there is a `<main />` tag in the admin pages. It was moved elsewhere is all. https://github.com/forem/forem/pull/10015#discussion_r479418367

Reply:
got it, thanks!